contrib/timpi

```diff
@@ -1 +1 @@
-Subproject commit 7fbe63d48e9d3ddee41b16935cdf172d3c604eec
+Subproject commit b38d5710d488181fcb3cb6e044f775f12f971000
```

Comment:
**Caution!** This contains a submodule update

Reply:
@milljm the new warning works, thanks!